css/about.css

```diff
@@ -0,0 +1,163 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+@font-face {
+  font-family: light-Lato;
+  src: url(/fonts/Lato2OFLWeb/Lato/fonts/Lato-Medium.ttf);
+}
+
+@media only screen and (max-width: 768px) {
+  .about {
+    display: flex;
+    flex-direction: column;
+    padding: 50px 10px;
+    width: 100%;
+    background: url(/image-resources/texture.jpg), #f5f5f5;
+    background-size: cover;
+    font-family: light-Lato, sans-serif;
+  }
+
+  .about-head {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-head h1 {
+    line-height: 25px;
+    font-family: bold-Lato, sans-serif;
+    background-image: linear-gradient(to right, #ec5242, #ec5242, #272a31, #ec5242, #ec5242, #ec5242);
+    /* stylelint-disable */
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    background-clip: text;
+    /* stylelint-enable */
+    letter-spacing: 5px;
+    font-weight: 900;
+    color: transparent;
+    font-size: 24px;
```

Comment:
- Great Job for fixing most request changes, you missed one thing the title `letter-spacing`, `font-size`, and `font-style`...  are different from the given design for both home and about page, please fix it as the previous reviewer suggested you

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/225645410-6a74f8a9-2592-4bcd-ada8-c7f8906bd058.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/225645419-3c350f73-079c-424a-81da-d10157e5db0b.png" width="200">|


😎️

🤝️